kernelci.org/content/en/docs/instances/staging.md

```diff
@@ -35,7 +35,7 @@ A special feature of the staging instance is the ability to test code from open
 GitHub pull requests before they get merged.  This is handled by tools in the
 [`kernelci-deploy`](https://github.com/kernelci/kernelci-deploy) project, to
 pull all the open pull requests for a given project, apply some arbitrary
-patches and push a resulting `staging.kernelci.org` branch back to the
+patches and push a resulting `staging-mainline` branch back to the
```

Comment:
It really is called `staging.kernelci.org`:
https://github.com/kernelci/kernelci-core/commits/staging.kernelci.org

The Linux kernel tree used on staging has a `staging-mainline` branch, as well as `staging-next` and `staging-stable` but that's not the same. This paragraph is about open PRs for KernelCI code, not the kernel repository.